variants/lionbits3/pins_arduino.h

```diff
@@ -51,13 +51,13 @@ static const uint8_t LDR = 7;
 static const uint8_t RGBLED = 48;
 
 // Analog to Digital Converter (Support 5V) ADC2 pins not recommended while using Wifi
-static const uint8_t A0  = 2;   //I/O,  GPIO2,    TOUCH2, ADC1_CH1, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM 
-static const uint8_t A1  = 1;   //I/O,  GPIO1,    TOUCH1, ADC1_CH0, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t A2  = 3;   //I/O,  GPIO3,    TOUCH3, ADC1_CH2, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM 
-static const uint8_t A3  = 4;   //I/O,  GPIO4,    TOUCH4, ADC1_CH3, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t A4  = 5;   //I/O,  GPIO5,    TOUCH5, ADC1_CH4, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM  
-static const uint8_t A5  = 6;   //I/O,  GPIO6,    TOUCH6, ADC1_CH5, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t AD1 = 7;   //I/O,  GPIO7,    TOUCH6, ADC1_CH5, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
+static const uint8_t A0  = 2;   //IO,  GPIO2,    TOUCH2, ADC1_CH1, */MAX 5V
+static const uint8_t A1  = 1;   //IO,  GPIO1,    TOUCH1, ADC1_CH0, */MAX 5V  
```

Comment:
@P-R-O-C-H-Y should the `MAX 5V` comment be kept here? I've not found any references from any of the datasheets indicating any voltage in excess of ~3.6v is safe for any IO pin.

Reply:
I agree, the 5V comment don't make any sense.

Reply:
Thanks @atanisoft @P-R-O-C-H-Y